nservicebus/upgrades/gateway-2to3.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Gateway Upgrade Version 2 to 3
+summary: Instructions on how to upgrade the Gateway from Version 2 to 3.
+component: Gateway
+related:
+ - nservicebus/upgrades/6to7
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 6
+ - 7
+---
+
+## Code first API
+
+The gateway now allows full configuration via code. Use `endpointConfiguration.Gateway()...` to enable and confgure sites and channels instead of `endpointConfiguration.EnableFeature<Gateway>()`.
```

Comment:
there should be before+after snippets here

Reply:
Snippets added